minitest/use_cases/user.py

```
@@ -0,0 +1,27 @@
+name_repo = NameRepository()
+
+def get_user(idx=None):
+    users = name_repo.get_name(idx)
+    return users
+
+def create_user(name):
+    user = User(name=name, created_at=datetime.now())
+    try:
+        user.validate() 
+    except ValueError as e:
+        raise ValidationError(str(e))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)